### PR TITLE
Integrate "User Password" within User Manager menu

### PR DESCRIPTION
**Pull request by Augustin-FL**
- [x] Redmine Issue: https://redmine.pfsense.org/issues/9428
- [x] Ready for review

### CODE REVIEW

File: src/usr/local/www/system_authservers.php [hunk 1]
```diff
@@ -453,7 +453,11 @@ function build_radiusnas_list() {
 }
 
 $tab_array = array();
-$tab_array[] = array(gettext("Users"), false, "system_usermanager.php");
+if (!isAllowedPage("system_usermanager.php*")) {
```
Comment (jim-p) [inline src/usr/local/www/system_authservers.php]: This doesn't need the `*` after the page name.

File: src/usr/local/www/system_groupmanager.php [hunk 2]
```diff
@@ -334,7 +334,11 @@ function build_priv_table() {
 }
 
 $tab_array = array();
-$tab_array[] = array(gettext("Users"), false, "system_usermanager.php");
+if (!isAllowedPage("system_usermanager.php*")) {
```
Comment (jim-p) [inline src/usr/local/www/system_groupmanager.php]: This doesn't need the `*` after the page name.

File: src/usr/local/www/system_groupmanager_addprivs.php [hunk 3]
```diff
@@ -137,7 +137,11 @@ function get_root_priv_item_text() {
 }
 
 $tab_array = array();
-$tab_array[] = array(gettext("Users"), false, "system_usermanager.php");
+if (!isAllowedPage("system_usermanager.php*")) {
```
Comment (jim-p) [inline src/usr/local/www/system_groupmanager_addprivs.php]: This doesn't need the `*` after the page name.

File: src/usr/local/www/system_usermanager_settings.php [hunk 4]
```diff
@@ -180,7 +180,11 @@
 }
 
 $tab_array = array();
-$tab_array[] = array(gettext("Users"), false, "system_usermanager.php");
+if (!isAllowedPage("system_usermanager.php*")) {
```
Comment (jim-p) [inline src/usr/local/www/system_usermanager_settings.php]: This doesn't need the `*` after the page name.

**Comment by Augustin-FL** [comment]
Done

**Pull request merged**